### PR TITLE
Add Polygon-Whitelist-NFT

### PR DESCRIPTION
**Pull request by neila**
## 変更内容
204 - Polygon-Whitelist-NFTの追加

### CODE REVIEW

**Review by yk-saito — COMMENTED** [review]
Hi, @wufengtao1 .

I am currently translating this into Japanese, but there are 4 things I would like to confirm.
I have left comments, so please check them and correct them if necessary!

**Comment by wufengtao1** [comment]
@yk-saito Thank you for your thorough review. Indeed, the amount of changes made previously was quite extensive, which led to some issues. I have now made corrections based on your feedback and have resubmitted a new pull request. Please take a look.
https://github.com/unchain-tech/UNCHAIN-projects/pull/440

File: docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-0/lesson-2_Setup MetaMask.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Click "Connect wallet" on the right side of the screen, select "Injected Web3 Provider," and then click on MetaMask to connect the wallet (Polygon Mainnet is the main network, while Mumbai is the testnet - we choose to Connect to Mumbai).  
+
+![image-20230114120433122](/public/images/Polygon-Whitelist-NFT_[ChainIDE]/section-0/0_2_4.png)
+
+
+#### Claim testnet tokens
+Once Mumbai is added to MetaMask, click on [Polygon Faucet](https://faucet.polygon.technology/) to receive testnet tokens. On the faucet page, we choose Mumbai as the network and MATIC as the token and then paste your MetaMask wallet address. Next, click submit, and the faucet will send you some test MATIC within a minute.
+
+<img src="/public/images/Polygon-Whitelist-NFT_[ChainIDE]/section-0/0_1_1.pngPolygon_PR_get_tokens.png" width="100%" height="100%" />
```
[the remark this answers is not in the document]
Comment (yk-saito) [reply docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-0/lesson-2_Setup MetaMask.md]: Solved by #440

File: docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-2/lesson-1_NFT Introduction.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+### NFT Protocol on ETH
+
+There are now many different NFT protocol standards on ETH, and let me introduce them to you in order:
+
+1. [ERC-721](https://eips.ethereum.org/EIPS/eip-721), one of the earliest NFT standards on Ethereum, was released in 2018. ERC-721 stands for "Non-Fungible Token," a unique digital asset. Each ERC-721 token has a distinct identifier (Token ID) to differentiate various tokens. ERC-721 tokens can be used to represent digital art, virtual items in games, collectibles, and more.
+
+2. [ERC-1155](https://eips.ethereum.org/EIPS/eip-1155) is a new NFT standard, released by Enjin in 2018. ERC-1155 represents "Fungible/Non-Fungible Tokens," merging both fungible and non-fungible tokens into one standard. This standard allows the creation of a single smart contract to manage multiple types of tokens, which can be either fungible or non-fungible. ERC-1155 tokens can be used to represent digital items, such as weapons, equipment, ornaments in games, and so on.
+
+3. Subsequent NFT protocols such as [ERC-3525](https://eips.ethereum.org/EIPS/eip-3525), [ERC-4907](https://eips.ethereum.org/EIPS/eip-3525) were also developed from the above two NFT standards.
```
[the remark this answers is not in the document]
Comment (yk-saito) [reply docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-2/lesson-1_NFT Introduction.md]: Solved by #440

File: docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-4/lesson-2_Build Frontend.md [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+
+### Code generation
+
+Create a new `Frontend` folder in the root directory on the left.
+
+![image-20230223143555437](/public/images/Polygon-Whitelist-NFT_[ChainIDE]/section-4/4_2_1.png)
+
+Create a new "index.html` file within the `frontend` directory.
+
+![image-20230223143709220](/public/images/Polygon-Whitelist-NFT_[ChainIDE]/section-4/4_2_2.png)
```
[the remark this answers is not in the document]
Comment (yk-saito) [reply docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-4/lesson-2_Build Frontend.md:11]: Solved by #440

File: docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-4/lesson-2_Build Frontend.md [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+
+Don't rush, we still need to fill in some information.
+
+```javascript
+ const contractAddress = ;
+```
+
+Here, you need to fill in your `shield` contract address, which you can copy from the `Deploy` panel.
+
+![image-20230223155653780](/public/images/Polygon-Whitelist-NFT_[ChainIDE]/section-4/4_2_3.png)
```
[the remark this answers is not in the document]
Comment (yk-saito) [reply docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-4/lesson-2_Build Frontend.md:192]: Solved by #440

File: docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-0/lesson-2_Setup MetaMask.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+### Setup MetaMask wallet
+
+#### Install MetaMask
+
+When we deploy a smart contract to the blockchain or interact with a deployed smart contract, we need to pay for gas. Therefore, we need a Web3 wallet, such as MetaMask. Click here to install MetaMask.[here](https://metamask.io/)
+
+#### Add Polygon Mumbai to MetaMask
+
+Polygon is a decentralized Ethereum Layer 2 blockchain that enables developers to build scalable, user-friendly dApps with low transaction fees without sacrificing security. Major NFT platforms such as Opensea and Rarible also support the Polygon Mumbai testnet, so we choose Mumbai to deploy our smart contract.
+
```
Comment (neila) [inline docs/Polygon-Whitelist-NFT_[ChainIDE]/en/section-0/lesson-2_Setup MetaMask.md:12]: Insert ChainIDE introduction and recommended use

**Comment by neila** [comment]
The project title should be standardized as `Polygon-Whitelist-NFT`. Meanwhile, make sure to provide sufficient room in section 1 to introduce and explain ChainIDE.

**Comment by neila** [comment]
metadata
```json
"project_id": 201,
"title": "Polygon_Whitelist_NFT",
"difficulty": "intermediate",
"chain": "Polygon",
"description": "👉 ブロックチェーン開発に特化した最新のコードエディタを用いながら、Polygon 上で指定のユーザーのみが NFT を発行できるdAppを開発します。"
"total_sections": CIに準拠
"total_lessons": CIに準拠
```

**Comment by wufengtao1** [comment]
@neila Hi, sho. If there's anything that needs to be done that requires Chainide's completion, please @ me :)

File: docs/Polygon-Whitelist-NFT/ja/section-1/lesson-2_基本的なコーディング.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+}
+```
+
+コードを一行ずつ解説します。
+
+```solidity
+// SPDX-License-Identifier: UNLICENSED
+```
+
+この行は`SPDXライセンス識別子`と呼ばれ、その後に続くコードの著作権の問題に対処します。一般的には、`UNLICENSED`と`MIT`が最もよく使われる接尾辞で、これは以下のコードが誰でも使ってよいことを示します。言い換えれば、皆さんは自由にコピー&ペーストすることができます。詳細は[こちら](https://spdx.org/licenses/?utm_source=buildspace.so&utm_medium=buildspace_project)で確認できます。
```
Comment (neila) [inline docs/Polygon-Whitelist-NFT/ja/section-1/lesson-2_基本的なコーディング.md:34]: remove query parameters from URL

Comment (yk-saito) [reply docs/Polygon-Whitelist-NFT/ja/section-1/lesson-2_基本的なコーディング.md]: fixed https://github.com/unchain-tech/UNCHAIN-projects/pull/439/commits/7475fdb89c42494dca6193ff68f47e1d3cbba36f

**Review by yk-saito — APPROVED** [review]

**Pull request merged**